### PR TITLE
Remove extra space for collection search

### PR DESCRIPTION
**Pull request by VivianChu**

### CODE REVIEW

**Comment by VivianChu** [comment]
@ucsdlib/developers - Please review, comment, etc...  Thanks

File: spec/features/dams_unit_spec.rb [hunk 1]
```diff
@@ -92,5 +97,11 @@
     
     # does not find this collection because it belongs to other unit
     expect(page).to_not have_selector('a', :text => 'Test Collection')
-  end  
+  end
+  
+  scenario 'rdcpt collection search' do
+    visit dams_unit_collections_path('rdcpt')
```
Comment (mcritchlow) [inline spec/features/dams_unit_spec.rb]: The overall space change seems 👍 

While the code for this test unit is indeed `rdcpt`, is that a description of the scenario being tested here? It seems to be about testing the contextual search display text. "Search and browse contextually within a unit", perhaps, or something like that. Curious what others think.

Comment (lsitu) [inline spec/features/dams_unit_spec.rb]: @mcritchlow Using more general contextual search display text like "Search and browse contextually within a unit" sounds good to me. Are you suggesting using this contextual search display text just for the test unit here, or it should be displayed on the search box for any units?

**Comment by VivianChu** [comment]
@mcritchlow Your suggestion sounds good to me.  The description has been updated.  Thanks.

**Comment by mcritchlow** [comment]
@VivianChu 👍 

@lsitu - I was just suggesting a potential change to the description of the test so that someone coming in to look at it for the first time (or perhaps a month or two from now) would be able to immediately get a sense of its purpose.

**Comment by lsitu** [comment]
👍 
@mcritchlow It sounds good to me.

**Comment by hweng** [comment]
👍

**Comment by hweng** [comment]
@VivianChu Can you squash the commits? Then I will merge it for you.

**Comment by VivianChu** [comment]
@hweng The commits have been squashed. Could you please merge it?  Thanks

**Pull request merged**